src/deny-list/time.ts

```diff
@@ -0,0 +1,20 @@
+
+// Number of milliseconds in one hour
+const MILLISECONDS_IN_HOUR = 60 * 60 * 1000;
+
+// Number of milliseconds in one day
+const MILLISECONDS_IN_DAY = 24 * MILLISECONDS_IN_HOUR;
+
+// Number of milliseconds from the current time to 2 AM UTC
+const MILLISECONDS_TO_2AM = 2 * MILLISECONDS_IN_HOUR;
+
+export const getIpListTTL = (time?: number) => {
+  const now = time ?? Date.now();
```

Comment:
Btw, if that time is `0` now will be Date.now() due to `??` not capturing if the value is 0. We might wanna replace this wanna `||` to switch to `Date.now()` whenever left side is falsy.